src/Command.hpp

```diff
@@ -35,6 +36,33 @@ class Command {
   unsigned long m_timeout;
 };
 
+class RecurrentCommand : public Command {
+ public:
+  explicit RecurrentCommand(const Command& command)
```

Comment:
it it necessary to have all those constructors? I don't think we need them all and might decide to provide only one